src/test/java/ca/team2706/frc/robot/commands/drivebase/TankDriveWithJoystickTest.java

```diff
@@ -0,0 +1,98 @@
+package ca.team2706.frc.robot.commands.drivebase;
+
+import com.ctre.phoenix.CTREJNIWrapper;
+import com.ctre.phoenix.motorcontrol.SensorCollection;
+import com.ctre.phoenix.motorcontrol.can.MotControllerJNI;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonSRX;
+import com.ctre.phoenix.sensors.PigeonIMU;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import ca.team2706.frc.robot.config.Config;
+import ca.team2706.frc.robot.subsystems.DriveBase;
+import edu.wpi.first.wpilibj.AnalogInput;
+import edu.wpi.first.wpilibj.Joystick;
+import edu.wpi.first.wpilibj.Notifier;
+import edu.wpi.first.wpilibj.PWM;
+import edu.wpi.first.wpilibj.drive.DifferentialDrive;
+import mockit.Expectations;
+import mockit.Injectable;
+import mockit.Mocked;
+import mockit.Verifications;
+import util.Util;
+
+public class TankDriveWithJoystickTest {
+    @Mocked
+    private WPI_TalonSRX talon;
+
+    @Mocked
+    private PWM pwm;
+
+    @Mocked
+    private AnalogInput analogInput;
+
+    @Mocked(stubOutClassInitialization = true)
+    private PigeonIMU pigeon;
+
+    @Mocked
+    private DifferentialDrive differentialDrive;
+
+    @Mocked(stubOutClassInitialization = true)
+    private CTREJNIWrapper jni;
+
+    @Mocked(stubOutClassInitialization = true)
+    private MotControllerJNI motControllerJNI;
+
+    @Mocked
+    private Notifier notifier;
+
+    @Injectable
+    private SensorCollection sensorCollection;
+
+    @Injectable
+    private Joystick joy1;
+
+    @Injectable
+    private Joystick joy2;
+
+    @Before
+    public void setUp() throws NoSuchFieldException, IllegalAccessException {
+        Util.resetSubsystems();
```

Comment:
You might want to do this in an `@BeforeClass` method just like I mentioned earlier, because this way it's only called once because in this case it only needs to be called once.
```suggestion
```